lib/ansible/modules/network/bigswitch/bigmon_policy.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Ansible module to manage Big Monitoring Fabric service chains
+# (c) 2016, Ted Elhourani <ted@bigswitch.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: bigmon_policy
+short_description: Create and remove a bigmon out-of-band policy.
+description:
+    - Create and remove a bigmon out-of-band policy.
+version_added: "2.3"
+options:
+  name:
+    description:
+     - The name of the policy.
+    required: true
+  policy_description:
+    description:
+     - description of policy.
+  action:
+    description:
+     - Forward matching packets to delivery interfaces, Drop is for measure rate of matching packets,
+       but do not forward to delivery interfaces, capture packets and write to a PCAP file, or enable NetFlow generation.
+    default: forward
+    choices: ['forward', 'drop', 'flow-gen']
+  priority:
+    description:
+     - A priority associated with this policy. The higher priority policy takes precedence over a lower priority.
+    default: 100
+  duration:
+    description:
+     - Run policy for duration duration or until delivery_packet_count packets are delivered, 
+       whichever comes first.
+    default: 0
+  start_time:
+    description:
+     - Date the policy becomes active
+    default: ansible_date_time.iso8601
+  delivery_packet_count:
+    description:
+     - Run policy until delivery_packet_count packets are delivered.
+    default: 0
+  state:
+    description:
+     - Whether the policy should be present or absent.
+    default: present
+    choices: ['present', 'absent']
+  controller:
+    description:
+     - The controller address.
+    required: true
+  validate_certs:
+    description:
+     - If C(false), SSL certificates will not be validated. This should only be used
+       on personally controlled devices using self-signed certificates.
+    required: false
+    default: true
+    choices: [true, false]
+  access_token:
+    description:
+     - Bigmon access token.
```

Comment:
`Bigmon access token. If this isn't set the the environment variable C(BIGSWITCH_ACCESS_TOKEN) is used.`